### DISCUSSION TITLE
How to deplay on heroku or replit ,or other free service?

### DISCUSSION THREAD

**Discussion by 63e** [searx/searx#3147] [General]
for i have no money and poor ,but still want my privacy
Please make it easy for green hand 
Thanks a lot

**Comment by 6s6t** [comment]
agreed

**Comment by unixfox** [comment]
It's not allowed by Heroku: https://github.com/searx/searx/issues/1631#issuecomment-508918576

**Reply by 63e** [reply]
How about others?

**Comment by 63e** [comment]
is it OK?
